### ISSUE TITLE
Question: Cannot login: Cannot regenerate session id - session is not active

### ISSUE THREAD

**Issue by Paranthaman-k** [delight-im/PHP-Auth#97]
We recently updated to PHP 7.2.2 from 5.4.29. We use redis for our session management. I face a problem  that I am unable to login. I see the below error:
 root.WARNING: E_WARNING: session_regenerate_id(): Cannot regenerate session id - session is not active {"code":2,"message":"session_regenerate_id(): Cannot regenerate session id - session is not active"

The above warning vanishes if I make an entry in /etc/hosts file in my local machine.

Could you please let me know how this affects the PHP?

Thanks in advance
Paranthaman

**Comment by Paranthaman-k** [comment]
Hi Marco,
Thanks a lot for your response. I completely understand what you mean. But it was my mistake I reported it as soon as I felt it was a strange behavior and did not take time to confirm the behavior before reporting. When I tried reproducing I was unable to. Sorry for the inconvenience caused. I am not sure what solved the issue. So I had to simply close. In case I could be successful in reproducing the same I will Reopen and post the solution.
Thanks!
